metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -1632,4 +1632,24 @@ After running `datahub ingest -c <path_to_recipe>`, our MCEs will now have the f
 ],
 ```
 
+### Using this in the remote executor (DataHub Cloud only)
+
+Build the image with your transformer
```

Comment:
I would add a line or two explaining what's in metadata-ingestion/examples/transforms/example.Dockerfile metadata-ingestion/examples/transforms and how to use it.